datadog_checks_dev/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - Datadog Checks Dev
 
+## 0.26.0 / 2019-06-01
+
+* [Added] Better error message when releasing on the wrong branch. See [#3832](https://github.com/DataDog/integrations-core/pull/3832).
```

Comment:
Should likely be a bugfix

Reply:
It was already explicit, this just made it a bit better

Reply:
I just meant we should probably bump the bugfix instead of the minor here. But either works 😄